calculator/src/App.js

```diff
@@ -0,0 +1,69 @@
+import React, { Component } from "react";
+import "./App.css";
+
+class App extends Component {
+  state = { textArray: [], sentences: [], numberE: [], commas: [] };
+
+  render() {
+    const handleSubmit = e => {
+      e.preventDefault();
+      console.log(e.target.text.value);
```

Comment:
A trick I like to use is command + f to check for any console.logs I may have accidentally left in my code.

Reply:
Eeek, thanks for that! I'll be sure to do that next time, time limit or no.